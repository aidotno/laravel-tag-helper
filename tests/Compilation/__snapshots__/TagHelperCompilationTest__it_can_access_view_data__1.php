<?php return '<div>
    <div>Some Information</div>
</div>';
