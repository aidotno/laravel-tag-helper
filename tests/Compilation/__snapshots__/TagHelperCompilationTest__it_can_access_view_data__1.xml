<?xml version="1.0"?>
<div>
  <div>Some Information</div>
</div>
